_posts/2025-03-05-next-scala-lts-jdk.md.md

```diff
@@ -0,0 +1,17 @@
+---
+category: announcement
+permalink: /news/next-scala-lts-jdk.html
+title: "JDK 17 will be the next JDK version used by Scala 3"
+by: Tomasz Godzik, VirtusLab & Scala Core Team
+---
+
+We recently asked for the community input about changing the JDK version used by
+the Scala 3 compiler. We wanted to thank everyone for their input and announce
+the decision to switch to JDK 17 starting with 3.8 minor and the next LTS which
+will mostly likely be Scala 3.9.
+
```

Comment:
Perhaps add something like: “This only concerns _future_ minor versions of Scala 3. In particular, the Scala 3.3 LTS line is unaffected by this change.“

Technically it's redundant to add but maybe it's worth being fully explicit, I don't know.

It could even continue to say, "Scala 3.3.x releases will continue for at least a year after the next LTS line begins," that kind of thing.

Reply:
Makes sense, it's better to be more explicit

Reply:
Added!